view/js/dropzone-factory.js

```diff
@@ -0,0 +1,65 @@
+var DzFactory = function () {
+	this.createDropzone = function(dropSelector, textareaSelector) {
+		return new Dropzone( dropSelector, {
+			paramName: 'userfile', // The name that will be used to transfer the file
+			maxFilesize: max_imagesize, // MB
+			url: '/media/photo/upload?response=url&album=',
+			addRemoveLinks: true,
+			acceptedFiles: 'image/*',
+			clickable: true,
+			accept: function(file, done) {
+				done();
+			},
+			init: function() {
+				this.on('success', function(file, serverResponse) {
+					const targetTextarea = document.getElementById(textareaSelector);
```

Comment:
I see you have changed the jQuery call to `getElementById`, so the parameter name should change as well
```suggestion
	this.createDropzone = function(dropSelector, textareaElementId) {
		return new Dropzone(dropSelector, {
			paramName: 'userfile', // The name that will be used to transfer the file
			maxFilesize: max_imagesize, // MB
			url: '/media/photo/upload?album=',
			addRemoveLinks: true,
			acceptedFiles: 'image/*',
			clickable: true,
			accept: function(file, done) {
				done();
			},
			init: function() {
				this.on('success', function(file, serverResponse) {
					const targetTextarea = document.getElementById(textareaElementId);
```

Reply:
yes. this was necessary to make setRangeText working as you suggested, that this won't work on jQuery-objects.